src/helix-ui/elements/_positioning.less

```diff
@@ -1,4 +1,4 @@
-@import '../vars.less';
+@import 'vars';
```

Comment:
Shadow Styles for positionable elements moved to `src/helix-ui/elements/` as a partial to include in each Shadow CSS.